src/pages/Market/Header/index.tsx

```diff
@@ -0,0 +1,68 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { useTranslation } from 'translation';
+import { formatCommaThousandsPeriodDecimal, convertCentsToDollars } from 'utilities/common';
+import { useStyles } from '../styles';
+
+interface IHeaderProps {
+  totalSupply: BigNumber;
+  totalBorrow: BigNumber;
+  availableLiquidity: BigNumber;
+  totalTreasury: BigNumber;
+}
+
+const Header: React.FC<IHeaderProps> = ({
+  totalSupply,
+  totalBorrow,
+  availableLiquidity,
+  totalTreasury,
+}) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  return (
+    <Paper css={styles.headerRoot}>
+      <div css={styles.row}>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.totalSupply')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {`$ ${formatCommaThousandsPeriodDecimal(
+              convertCentsToDollars(totalSupply.toNumber()),
+            )}`}
+          </Typography>
+        </Paper>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.totalBorrow')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCommaThousandsPeriodDecimal(convertCentsToDollars(totalBorrow.toNumber()))}
+          </Typography>
+        </Paper>
+        <Paper css={styles.box}>
+          <Typography variant="small1" css={styles.title}>
+            {t('market.availableLiquidity')}
+          </Typography>
+          <Typography variant="h3" css={styles.value}>
+            {formatCommaThousandsPeriodDecimal(
+              convertCentsToDollars(availableLiquidity.toNumber()),
+            )}
```

Comment:
Same comment about using `formatCentsToReadableValue`

Reply:
I chose not to use formatCentsToReadableValue because the designs do not show a dollar sign